File: backend/src/main/java/com/backend/ord/domain/dto/QuicklyAddedWordDTO.java
package com.backend.ord.domain.dto;

import com.backend.ord.domain.dto.abstracts.DTOBase;
import com.backend.ord.enums.Language.LanguageName;
import lombok.*;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class QuicklyAddedWordDTO extends DTOBase {
    private String typedWord;
    private LanguageName typedInLanguage;

    private UserDTO user;
}

File: backend/src/main/java/com/backend/ord/domain/dto/QuicklyAddedWordDTO.kt
package com.backend.ord.domain.dto

import com.backend.ord.domain.dto.abstracts.DTOBase
import com.backend.ord.enums.Language.LanguageName

data class QuicklyAddedWordDTO(
    var typedWord: String,
    var typedInLanguage: LanguageName,

    var user: UserDTO
) : DTOBase()
